budgetiser/lib/screens/categories/shared/editCategory.dart

```diff
@@ -0,0 +1,94 @@
+import 'package:budgetiser/screens/categories/shared/categoryForm.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_colorpicker/flutter_colorpicker.dart';
+
+class EditCategory extends StatefulWidget {
+  Color selectedColor;
+  String categoryTitle;
+  EditCategory({
+    Key? key,
+    required Color this.selectedColor,
+    required String this.categoryTitle,
+  }) : super(key: key);
+
+  @override
+  State<EditCategory> createState() => _EditCategoryState(selectedColor, categoryTitle);
+}
+
+class _EditCategoryState extends State<EditCategory> {
+  Color selectedColor = Colors.white;
+  String categoryTitle = "";
+  _EditCategoryState(Color selectedColor, String categoryTitle){
+    this.selectedColor = selectedColor;
+    this.categoryTitle = categoryTitle;
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: const Text("Edit Category"),
+      ),
+      body: Container(
+        alignment: Alignment.topCenter,
+        child: SingleChildScrollView(
+          child: Padding(
+            padding: const EdgeInsets.symmetric(horizontal: 8),
+            child: Column(
+              children: [
+                CategoryForm(initialName: categoryTitle),
+                InkWell(
```

Comment:
ich würd den colorpicker in nen eigenes file machen

Reply:
Ist mir tatsächlich auch eben aufgefallen. Wollte ich noch ändern heute. Wie viel würdest du davon auslagern? Also nur den ShowDialog aus OnTap oder auch das ganze Inkwell? Wenn nur der Dialog genommen wird könnte man den gleichen Picker wiederverwenden aber z.B. auch in anderen Formen/Größen

Reply:
fürs klickibunti ist der ganze inkwell einfacher, weil du dann die eingabe net an editCategory zurückgeben musst... 
sonst kann man dem ja noch ne width mitgeben oder so, weil ich denk des passt wenn der überall gleich ist